sync/wptfyi.py

```diff
@@ -104,22 +106,27 @@ def get_results(run_ids, test=None, query=None, staging=False):
     return resp.json()
 
 
-def get_metadata(products, link):
-    url = Url(WPT_FYI_BASE + "search")
+def get_metadata(products, link, staging=False):
+    url = Url(WPT_FYI_BASE + "metadata")
+    if staging:
+        url.host = STAGING_HOST
 
     if isinstance(products, basestring):
         url.add_query("product", products)
     else:
         for product in products:
             url.add_query("product", product)
-    if link:
-        body = {"exists": [{"link": link}]}
-    else:
-        body = None
+    resp = requests.get(url.build())
 
-    if body is not None:
-        resp = requests.post(url.build(), json=body)
```

Comment:
Do we no longer have this use case?

Reply:
This never worked the way it was supposed to; the `search` api was returning the wrong things. Now we just do the same thing in client code (but this will all go away once we have the metadata repo available locally).